dev-resources/template/2_lrs.yml

```diff
@@ -75,10 +52,7 @@ Parameters:
   InstanceSSHCidr:
     Description: CIDR Range for SSH Access to instances (Typically VPC CIDR Range)
     Type: String
-    Default: "173.147.0.0/16"
-  InstanceSubnets:
-    Description: Subnet on which to run the lrsql server
-    Type: List<AWS::EC2::Subnet::Id>
```

Comment:
Why is `InstanceSubnets` going away? I note `DBSubnets` is used instead, but per Cliff `DBSubnets' is actually the subnet the lambda `DBInitFn` runs on and might need to be different from that of the instances.